### PR TITLE
Fix deep merge of tools

### PR DESCRIPTION
**Pull request by hata6502**
https://github.com/codex-team/editor.js/issues/1017

### CODE REVIEW

**Comment by khaydarov** [comment]
@hata6502 hi and thanks! Can you describe the real case of passing a "very big object" to the config? I think it is better to throw an exception

File: src/components/modules/tools.ts [hunk 1]
```diff
@@ -221,7 +221,30 @@ export default class Tools extends Module {
     /**
      * Assign internal tools
      */
-    this.config.tools = _.deepMerge({}, this.internalTools, this.config.tools);
+    this.config.tools = {
+      ...this.internalTools,
+      ...this.config.tools,
+      bold: {
```
Comment (neSpecc) [inline src/components/modules/tools.ts]: I don't think it is good idea to hardcode internal tools here. It would better to store their names in one place

**Comment by hata6502** [comment]
https://github.com/codex-team/editor.js/pull/1018#issuecomment-576671763

In my case, I tried setting ApolloClient to config. 
https://github.com/apollographql/apollo-client

**Comment by hata6502** [comment]
Sorry for lack of information. 
I write more detail :)

In my case, I'm making my Image block. 
Apollo Client is used to get picture url. 
I tried to propagate Apollo Client via tools.config. 

I think deep merge should be used little, because it is high load. 
Especially internalTools's nest levels are fixed, so deep merge is not needed.

**Comment by hata6502** [comment]
This PR causes ToolConfig error when launched multiple EditorJS. 
So I close it.

**Pull request closed without merging**